clone/cmd/sumdbclone/sumdbclone.go

```diff
@@ -63,6 +64,24 @@ func main() {
 	client := sdbclient.NewSumDB(tileHeight, *vkey, *url, &http.Client{
 		Timeout: *timeout,
 	})
+	cloneAndVerify(ctx, client, db)
+	if *pollInterval == 0 {
+		return
+	}
+	ticker := time.NewTicker(*pollInterval)
+	for {
+		select {
+		case <-ticker.C:
```

Comment:
Why doesn't it call cloneAndVerify directly here?

Reply:
No good reason other than consistency with some other code I'd previously written. Agree that this way is easier to read. Fixed.